lib/plugins/aws/deploy/lib/iam-role-lambda-execution-template.json

```diff
@@ -16,6 +16,31 @@
           ]
         }
       ]
-    }
+    },
+    "Policies": [
+      {
+        "PolicyName": "[TO BE REPLACED]",
+        "PolicyDocument": {
+          "Version": "2012-10-17",
+          "Statement": [
+            {
+              "Effect": "Allow",
+              "Action": [
+                "logs:CreateLogGroup",
+                "logs:CreateLogStream"
+              ],
+              "Resource": []
+            },
+            {
+              "Effect": "Allow",
+              "Action": [
+                "logs:PutLogEvents"
+              ],
+              "Resource": []
+            }
+          ]
+        }
+      }
+    ]
```

Comment:
Makes total sense to update, especially if were ALWAYS creating the log groups. Will only keep `CreateLogStream` and `PutLogEvents` then.